pages/cloud/lake-effect.md

```diff
@@ -1,47 +1,71 @@
+Features of LakeEffect:
 
-- Deploy virtual servers in minutes
-- Storage volume creation/snapshot
-- VPC Networking with access to public IP address space as well as the ability to create private networks
-- Wide range of Machine Types available inclunding GPUs and IO Optimized Storage
-- Various Linux operating systems are supported.  _**NOTE**: Windows images are not provided be CCR at this time_
-- Public Images of opensource Linux Distributions available as well as the ability for users to upload their own
-- Load balancing of virtual resources across multiple physical machines
-- Support for Floating IP addresses
-- Easily manage access to VMs using Security Groups and Rules
-- Built on Openstack Cloud Software [Openstack](https://www.openstack.org/) 
-- Fault tolerant components to minimize risk of downtime due to underlying hardware failure
+* Deploy virtual servers in minutes
+* Storage volume creation/snapshot
+* VPC Networking with access to public IP address space as well as the ability to create private networks
+* Wide range of Machine Types available inclunding GPUs and IO Optimized Storage
+* Various Linux operating systems are supported.  _**NOTE**: Windows images are not provided be CCR at this time_
+* Public Images of opensource Linux Distributions available as well as the ability for users to upload their own
+* Load balancing of virtual resources across multiple physical machines
+* Support for Floating IP addresses
+* Easily manage access to VMs using Security Groups and Rules
+* Built on Openstack Cloud Software [Openstack](https://www.openstack.org/) 
+* Fault tolerant components to minimize risk of downtime due to underlying hardware failure
 
+!!! Warning
+    CCR does **not** backup LakeEffect Research Cloud. Backup of cloud data and configurations is the user’s responsibility!
 
 ### LakeEffect Resources
-> - 768 total CPU cores
-> - 8.8TB RAM (4GB per vCPU core)
-> - 1.7PB of object-oriented storage
-> - NVidia V100 vGPUs
-> - NVME Ephemeral Storage 
-> - Persistant Memory (PMEM) Storage 
+> * 768 total CPU cores
+> * 8.8TB RAM (4GB per vCPU core)
+> * 1.7PB of object-oriented storage
+> * NVidia V100 vGPUs
+> * NVME Ephemeral Storage 
+> * Persistant Memory (PMEM) Storage 
+
+
+### Subscriptions
+
+In order to use LakeEffect Research cloud, you will need to purchase a subscription. 
+
+
+| Package Name | vCPU | Storage | Other | Price |
+| :--- | ---: | ---: | :----: | ---:| 
+| Starter| 10 vCPU | 1 TB | 1 Floating IP | $558.81/year |
+| Standard| 20 vCPU | 2 TB | 2 Floating IP | $1117.62/year |
+| Advanced| 40 vCPU | 4 TB | 4 Floating IP , Fast-IO VMs | $2235.24/year |
+| GPU Addon| 8 vCPU | 1 TB | 1 Floating IP, Tesla V100 16GB | $936.34/year |
+| Storage addon | | 1 TB | Block Storage | $54.36/TB/year |
+
+* Systems Administration Consulting: $59.97/hour
+
+    * Must be purchased in 8 hour increments
+    * Can be used in 30 minute chunks
+    * Contact ccr-help for estimates on required time
```

Comment:
'Contact CCR Help' link to new help.md file to be created - here and in the tip below

Reply:
replaced all the ccr-help@buffalo.edu with a link to the help doc.